src/App.css

```diff
@@ -1,38 +1,72 @@
-.App {
+.container {
+  background-color: #E5E5E5;
+  height: 90%;
+  width: 96%;
+  margin: auto;
+  text-align: justify;
+}
+.bedTab {
+  width: 50%;
+  text-align: center;
+}
+.bedTabSelected {
+  background-color: grey;
+  width: 50%;
+  text-align: center;
+}
+.beds {
+  font-size: 36px;
+  
+}
+.ringDown {
+  font-size: 36px;
+}
+.ringDownTab {
+  width: 50%;
+  position: relative;
+  left: 27.79em;
+  top: -2.7em;
+  text-align: center;
+}
+.ringDownTabselected {
+  background-color: grey;
+  width: 50%;
+  position: relative;
+  left: 27.79em;
+  top: -2.7em;
   text-align: center;
 }
-
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
-}
-
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
-}
-
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
-}
-
-.App-link {
-  color: #61dafb;
-}
-
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
-  to {
-    transform: rotate(360deg);
-  }
+a:link {
+  color: black;
+}
+a:visited {
+  color: black;
+}
+.bedContent {
+  padding: 10% 10% 10% 10%;
+}
+.bedComponentContainer {
+  padding: 0% 10% 7% 3%;
+  border-bottom: 0.05px solid black;
+}
+a {
+  text-decoration: none;
+}
+.RingDownInfo {
```

Comment:
I see some differences in naming convention, i.e. lowercase "ringDownContainer" and here uppercase "RingDownInfo". Should we define a naming convention to follow for the team?

I'm not too strict about it in my projects, but I generally try to follow BEM style naming conventions:

http://getbem.com/

Reply:
Yes I think a defined naming convention would be great. I looked at the article you posted. I will use the example from that article. Thank you!